jkamz/ireporter
# #164326073 Configure confirm reset password url and write tests.

#### What does this PR do?
Has tests for password reset endpoint
#### Description of Task to be completed?
Configure `/api/auth/reset_password_confirm/` url and write tests for the endpoint.
#### How should this be manually tested?
1. Clone this repo.
   `git clone https://github.com/jkamz/ireporter.git`
2. Fetch all origin to get all branches
   `git fetch origin`
3. Checkout to `ft-reset-password-email-164326073` branch.
   `git checkout ft-reset-password-email-164326073`
4. Activate `virtualenv`
    `$ virtualenv -p python .venv`
    `$ source .venv/bin/activate`
5. Install requirements
   `$ pip install -r requirements.txt`
6. Start server
   `$ python api/manage.py runserver`
7. Test the endpoint on postman in the following steps:
      - `/api/auth/signup/`
      - `/api/auth/activate/`
      - `/api/auth/reset_password/`
      - `/api/auth/reset_password_confirm/`
(Ensure to copy the generated token and uid generated in the `/api/auth/reset_password_confirm?uid=string_uid&token=generated_token` url)
8. Run tests:
   `$ python api/manage.py test`
#### What are the relevant pivotal tracker stories?
[#164326073](https://www.pivotaltracker.com/story/show/164326073)
#### Screenshot
![image](https://user-images.githubusercontent.com/17095461/53999182-92a6b280-4153-11e9-97b3-ef604fc583ae.png)
